### ISSUE TITLE
Very weird " bloody " effect on my i2v wan 2.2 lora's then epoch is high

### ISSUE THREAD

**Issue by BenDes21** [kohya-ss/musubi-tuner#626]
Hi there, currently training a motion i2v wan 2.2 lora, Im pretty satisfied about the motion Im getting from the last epochs but the issue is that  more the epoch is high ( so more my motion is accurate ) more Im getting a disgusting blood effect on my characters when I test the loras, it's very disturbing and frustrating because the motion seems correctly learned, I guess there is an issue with my config but for me it's seems ok, is anyone know what can cause the issue ? Thanks a lot

[[datasets]]
video_directory = "/workspace/motion"
cache_directory = "/workspace/motion"
caption_extension = ".txt"
enable_bucket = true

resolution = [256, 256]
target_frames = [81]

frame_extraction = "full"

high 

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 src/musubi_tuner/wan_train_network.py \
    --max_train_epochs 100 --save_every_n_epochs 10 --seed 42 \
    --output_dir /workspace/output --output_name motion_HIGH \
    --network_module networks.lora_wan \
    --task i2v-A14B \
    --dataset_config /workspace/dataset.toml \
    --min_timestep 900 --max_timestep 1000 \
    --dit /workspace/models/wan2.2_i2v_high_noise_14B_fp16.safetensors \
    --t5 /workspace/models/umt5-xxl-enc-bf16.safetensors \
    --vae /workspace/models/wan_2.1_vae.safetensors \
    --discrete_flow_shift 5.0 --timestep_sampling shift  \
    --sdpa --preserve_distribution_shape --gradient_checkpointing --force_v2_1_time_embedding \
    --fp8_base --fp8_scaled --mixed_precision fp16 \
    --max_data_loader_n_workers 32 --persistent_data_loader_workers \
    --network_module networks.lora_wan --network_dim 32 --network_alpha 32 \
    --optimizer_type schedulefree.RAdamScheduleFree --learning_rate 3e-5 

low : 

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 src/musubi_tuner/wan_train_network.py \
    --max_train_epochs 100 --save_every_n_epochs 10 --seed 42 \
    --output_dir /workspace/output --output_name motion_LOW \
    --network_module networks.lora_wan \
    --task i2v-A14B \
    --dataset_config /workspace/dataset.toml \
    --min_timestep 0 --max_timestep 900 \
    --dit /workspace/models/wan2.2_i2v_low_noise_14B_fp16.safetensors \
    --t5 /workspace/models/umt5-xxl-enc-bf16.safetensors \
    --vae /workspace/models/wan_2.1_vae.safetensors \
    --discrete_flow_shift 5.0 --timestep_sampling shift  \
    --sdpa --preserve_distribution_shape --gradient_checkpointing --force_v2_1_time_embedding \
    --fp8_base --fp8_scaled --mixed_precision fp16 \
    --max_data_loader_n_workers 32 --persistent_data_loader_workers \
    --network_module networks.lora_wan --network_dim 32 --network_alpha 32 \
    --optimizer_type schedulefree.RAdamScheduleFree --learning_rate 3e-5 

effect ( not gonna lie this one was a bit nsfw so I just show a small portion of the effect )  : 

epoch 70
<img width="153" height="158" alt="Image" src="https://github.com/user-attachments/assets/e5d5ac6a-f5e3-4db3-ad69-49002e456a14" />

epoch 80
<img width="209" height="154" alt="Image" src="https://github.com/user-attachments/assets/7cf262c0-fe74-424e-ae52-429c3123917d" />

epoch 90

<img width="230" height="152" alt="Image" src="https://github.com/user-attachments/assets/d4245224-8f5d-4dc1-84f5-0ab90be81246" />

**Comment by kohya-ss** [comment]
If the training data does not contain such effects, I have no idea...

Is it possible to suppress this with negative prompts?
